website/content/roadmap.md

```diff
@@ -75,7 +75,7 @@ because fast iteration cycle can drastically decrease frustrations caused by
 banal misspellings.
 
 This requires development across multiple dimensions — writing an
-[LSP server](https://langserver.org/), better support for typing in the
+[LSP server,](https://langserver.org/) better support for typing in the
```

Comment:
Surely the comma shouldn't be part of the link? It's a separator?

Reply:
This avoids a particularly annoying line break in my browser when the window is wide. (See image below.) This is my solution to a long-standing conundrum: placing closing punctuation (commas, periods) inside the `<a> ... </a>` tag to avoid this problem.

<img width="1378" alt="image" src="https://user-images.githubusercontent.com/1094930/219883142-06a64dd4-6710-41e2-8159-f71d37105b1c.png">

Reply:
I think this is caused by the space being added after the external link token, which IIRC happens because the autoformatter adds a newline at the end of the file which adjusts external links.

The solution would be to fix that...

Reply:
It looks like this on my screen:

![image](https://user-images.githubusercontent.com/11072061/219965216-34722a1e-9b23-48ae-b5f5-559487659596.png)